File: class15/python/AP1510-ClassificacaoIdadeDETALHADA.py
# ClassificacaoIdadeDETALHADA — Aula 15
# Classifica faixa etária com escolha/caso (simulado com if-elif)

idade = int(input("Idade: "))
if idade < 0:
    print("Idade inválida!")
elif idade <= 12:
    print("Criança")
elif idade <= 17:
    print("Adolescente")
elif idade <= 59:
    print("Adulto")
elif idade <= 120:
    print("Idoso")
else:
    print("Idade improvável!")

File: class15/cpp/AP1510-ClassificacaoIdadeDETALHADA.cpp
#include <iostream>
int main() {
    int idade;
    std::cout << "Idade: "; std::cin >> idade;
    if (idade < 0) std::cout << "Idade inválida!" << std::endl;
    else if (idade <= 12) std::cout << "Criança" << std::endl;
    else if (idade <= 17) std::cout << "Adolescente" << std::endl;
    else if (idade <= 59) std::cout << "Adulto" << std::endl;
    else if (idade <= 120) std::cout << "Idoso" << std::endl;
    else std::cout << "Idade improvável!" << std::endl;
    return 0;
}